brigadier/src/main/java/co.aikar.commands/ACFBrigadierManager.java

```diff
@@ -89,6 +89,12 @@
         root = rootBuilder.build();
         boolean isForwardingCommand = rootCommand.getDefCommand() instanceof ForwardingCommand;
 
+        if (rootCommand.getDefaultRegisteredCommand() != null) {
+            CommandNode<S> paramNode = root;
+            CommandParameter[] parameters = rootCommand.getDefaultRegisteredCommand().parameters;
+            registerParameters(parameters, rootCommand, paramNode, suggestionProvider, executor, permCheckerRoot);
```

Comment:
Why do you pass the rootCommand and permCheckerRoot here? pretty sure it's enough to pass `rootCommand.getDefaultRegisteredCommand()` and `permCheckerSub`.

Your current implementation will provide wrong suggestions for @Default command methods with a @CommandPermission.